src/Appwrite/Specification/Format.php

```diff
@@ -21,43 +21,56 @@ abstract class Format
      */
     protected array $models;
 
+    /**
+     * @var Event []
+     */
+    protected array $events;
+
     protected array $services;
     protected array $keys;
     protected int $authCount;
     protected array $params = [
-        'name' => '',
-        'description' => '',
-        'endpoint' => 'https://localhost',
-        'version' => '1.0.0',
-        'terms' => '',
-        'support.email' => '',
-        'support.url' => '',
-        'contact.name' => '',
-        'contact.email' => '',
-        'contact.url' => '',
-        'license.name' => '',
-        'license.url' => '',
+        "name" => "",
+        "description" => "",
+        "endpoint" => "https://localhost",
+        "version" => "1.0.0",
+        "terms" => "",
+        "support.email" => "",
+        "support.url" => "",
+        "contact.name" => "",
+        "contact.email" => "",
+        "contact.url" => "",
+        "license.name" => "",
+        "license.url" => "",
```

Comment:
We typically use single quote for strings.

Reply:
@stnguyen90 I ran the formatter lol I was hoping this would work :P

Reply:
Fixed